vdev.c

```diff
@@ -1940,7 +1940,8 @@ static int timer_read(void *opaque)
     uint64_t count;
 
     /* Read the count to rearm the periodic timer, but ignore the result */
```

Comment:
This explicitly says to _ignore_ the result. Have you verified that asserting on it doesn't break anything?